spec/models/file_set/file_set_derivatives_spec.rb

```diff
@@ -5,7 +5,7 @@
   let(:file)        { build(:department_file, id: '1234') }
   let(:derivatives) { CurationConcerns::DerivativePath.derivatives_for_reference(file) }
 
-  before { LakeshoreTesting.reset_derivatives }
+  before(:each) { LakeshoreTesting.reset_derivatives }
```

Comment:
@jenlindner I think this may be related to our intermittent failures. Let me take a look.